src/components/RadioField/RadioField.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Input } from './style';
+
+export default function RadioField(props) {
+  const {
+    error, onChange, options,
+  } = props;
+  return (
+    <>
+      { options && options.length && options.map(({ value, label }) => (
```

Comment:
use default blank array in default propTypes for _options_ field and remove the _&&_ conditions